config/webpack.config.dev.js

```diff
@@ -0,0 +1,24 @@
+const merge = require('webpack-merge');
+const path = require('path');
+const Dotenv = require('dotenv-webpack');
+const commonConfig = require('./webpack.config.common');
+
+module.exports = merge(commonConfig, {
+  mode: 'development',
+  devtool: 'eval-source-map',
+
+  devServer: {
+    contentBase: path.join(__dirname, '../build'),
```

Comment:
why is `build` directory the contentBase?, are we going to be building always in development?

Reply:
Thank you @dinobi I will change it to src since we won't be building in the development area.